py_ocsf_models/events/findings/finding.py

```diff
@@ -79,6 +79,7 @@ class FindingInformation(BaseModel):
     Attributes:
     - Analytic (analytic) [Recommended]: The analytic technique used to analyze and derive insights from the data or information that led to the finding or conclusion.
     - Created Time (created_time) [Optional]: The time when the finding was created.
+    - Created Time DT (created_time_dt) [Optional]: The time when the finding was created in datetime format.
```

Comment:
Please add the rest of the new fields here.

Reply:
Done!